inventory_price_check/models/stock_picking.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class Stockpick(models.Model):
+    _inherit = "stock.picking"
+
+    allow_price_inconsistency = fields.Boolean(
+        copy=False,
+        tracking=True,
+        groups="inventory_price_check.group_purchase_inventory_discrepancy",
+        help="If enabled, no error is raised for price inconsistency between "
+        "recieved price and standard price",
+    )
+
+    def _action_done(self):
+        for pick in self:
+            if pick.picking_type_id.code != "incoming":
+                continue
+            if pick.sudo().allow_price_inconsistency:
```

Comment:
Why `sudo()` here?

Reply:
This field is only allowed for some group and we will get access right error if the user is not in this group.